x-pack/plugins/graph/public/types/app_state.ts

```diff
@@ -8,7 +8,7 @@
 import { SimpleSavedObject } from 'src/core/public';
 import { FontawesomeIcon } from '../helpers/style_choices';
 import { OutlinkEncoder } from '../helpers/outlink_encoders';
-import type { IndexPattern } from '../../../../../src/plugins/data/public';
+import type { DataView } from '../../../../../src/plugins/data/common';
```

Comment:
Could you import all `DataView` related logic from /data_views/public or /data_views/server, thx